regexes/oss.yml

```diff
@@ -300,6 +300,18 @@
   version: ''
 
 
+##########
+# tvOS (https://developer.apple.com/tvos/)
+##########
+- regex: 'Apple TV.*CPU OS ((?:9|10|11|12|13|14)_\d+(?:[_\.]\d+)*)'
+  name: 'tvOS'
+  version: '$1'
+
+- regex: 'iOS; ((?:9|10|11|12|13|14).\d+(?:[_\.]\d+)*) AppleTV'
+  name: 'tvOS'
+  version: '$1'
```

Comment:
Isn't there still a AppleTV detection in that file? Guess that should be removed and covered with those detections

Reply:
It can't be removed because there are two different types of user agents, only if we can combine those two regexes.
`AppleCoreMedia/1.0.0.12B466 (Apple TV; U; CPU OS 8_1_3 like Mac OS X; en_us)
XBMC/PRE-11.0 Git:20110623-62171b3 (iOS; 11.0.0 AppleTV2,1; http://www.xbmc.org)`

Reply:
I was referring to this one:
https://github.com/matomo-org/device-detector/blob/cdf7d731e0d6b4ff2f8791f1debe6612649d4a0f/regexes/oss.yml#L685-L687

Reply:
here you can improve
```yml
- regex: 'iOS; ((?:9|1[0-4]).\d+(?:[_\.]\d+)*) AppleTV'
  name: 'tvOS'
  version: '$1'
  ```